cc/oak_echo_raw_enclave_app/BUILD

```diff
@@ -0,0 +1,14 @@
+cc_binary(
+    name = "oak_echo_raw_enclave_app",
+    srcs = ["main.cc"],
+    copts = [
+        "-nostdlib",
+        "-ffreestanding",
+    ],
+    features = ["fully_static_link"],
+    linkopts = [
+        "-nostdlib",
+        "-zmax-page-size=0x200000",
```

Comment:
Why do we need to specify this?

Reply:
* `-nostdib` because we're not using a cross-compiler and we don't want to link against libc from the host (we're providing newlib on our own)
* `-zmax-page-size` because the default SysV ABI expects page size to be 4 KiB, but in Restricted Kernel, thus far, we've used 2 MiB pages throughout